pkg/resources/testdata/TestAcc_View_basic/variables.tf

```diff
@@ -0,0 +1,31 @@
+variable "name" {
+  type = string
+}
```

Comment:
should the vairables also be moved into the test.tf file?

Reply:
What do you mean? For every test written this way, we have `test.tf` with config and `variables.tf` with variables.